mutable_tree_test.go

```diff
@@ -29,7 +29,7 @@ var (
 
 func setupMutableTree(t *testing.T) *MutableTree {
 	memDB := db.NewMemDB()
-	tree, err := NewMutableTree(memDB, 0)
+	tree, err := NewMutableTree(memDB, 0, false)
```

Comment:
we should add a couple tests that has it disabled. talked with dev and he mentioned a few more places are needed. The gets was one thing he pointed to

Reply:
ok, I would convert this to draft and continue working on other possible affected parts and adding some tests

Reply:
added two integration tests to make sure that the tree will not perform the fast storage upgrade when loading the tree, and the not upgraded tree returns correct key/value with `Get` and `Iterate`.